pkg/ebpf/c/vmlinux.h

```diff
@@ -788,6 +788,11 @@ struct iovec {
     __kernel_size_t iov_len;
 };
 
+enum
+{
+    BPF_F_NO_PREALLOC = 1,
```

Comment:
```suggestion
    BPF_F_NO_PREALLOC = (1U << 0),
```

As in the kernel code, making it crystal as a bitmask.